index.php

```diff
@@ -0,0 +1,111 @@
+<?php
+session_start();
+require 'connection.php';
+?>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="bootstrap.css">
+    <link rel="stylesheet" href="styles.css">
+</head>
+
+<body>
+    <div class="container-fluid">
+        <div class="row">
+            <?php
+            if (isset($_SESSION["user"])) {
+                $uID = $_SESSION["user"]->id;
+                include 'navbar-logged-in.php';
```

Comment:
Don't use 2 files (`navbar.php` and `navber-logged-in.php`) for navbar. You can manage this feature using only 1 file. You must program the navbar (use session management to validate the user in the `navbar.php` file) and include this file in `index.php`.

Reply:
OK